Makefile

```diff
@@ -44,7 +44,7 @@ codegen: gocodegen
 
 # Build the asciidoc book.
 .PHONY: docs
-docs:
+docs: generator
```

Comment:
Thanks @ebeahan `{source_branch}` looks promising, I'll see if I can make that work with less intervention likewise adding a new target

Reply:
@ebeahan Moving back to draft while I work on this - the fields `{source_branch}` and also `{version}` are already available so I should use those rather than building (potentially desynchronized) alternatives, however, their underlying logic is a little unclear to me: for example if I set the file 

`ecs/version` to `1.10.0`

and then generate the docs

then 

`version` evaluates to `1.8`

and

`source_branch` evaluates to `master` 

Which is not what I would have (initially) expected

I'm looking into the upstream bash scripts to see whether they need fixing or I have simply misinterpreted them  + evaluate whether any other code/docs relies on their current behavior

Reply:
Please see summary in https://github.com/elastic/ecs/pull/1338#issuecomment-824305976